services/jobrunner/install.sh

```diff
@@ -21,17 +21,7 @@ mkdir -p $TARGET_DIR
 test -d $TARGET_DIR/code || git clone https://github-proxy.opensafely.org/opensafely-core/job-runner $TARGET_DIR/code
 test -d $TARGET_DIR/lib || git clone https://github-proxy.opensafely.org/opensafely-core/job-runner-dependencies $TARGET_DIR/lib
 
-# service configuration
-SECRET_DIR=$HOME_DIR/secret
-mkdir -p $SECRET_DIR
-chmod 0700 $SECRET_DIR
-find $SECRET_DIR -type f -exec chmod 0600 {} \;
-
-BIN_DIR=$HOME_DIR/bin
-mkdir -p $BIN_DIR
-
-cp $SRC_DIR/bin/* $BIN_DIR/
-cp bin/lsjobs $BIN_DIR/
+cp $SRC_DIR/bin/* ~opensafely/bin/
```

Comment:
while we're doing a rename, should we also rename `lsjobs` to `lsjobs.sh`? (given that it's a shell script). We don't have to though.

Reply:
`lsjobs`i s intended to invocate manually as cli command, so does not have .sh deliberately, as that is unix convention AIUI.

The only other script in bin is update-jobrunner.sh, which is called by just rather than 
directly by users.

In this case, bin is action both as something on the PATH, and a place to put scripts, which is maybe not ideal, but a jobrunner/scripts dir seems overkill for one script?

Reply:
Am gonna punt on this for now, as I think my current WIP is gonna add a some more ./bin/ commands, so we can deal with this then.

Reply:
My current WIP is gonna impact on this, so punting for now.